apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -39,7 +39,8 @@ public class TestDatabaseFixture : IDisposable
         // not cause failures).
         // if you use a scratch database, be sure to comment out the database
         // creation and disposal methods.
-        public string DatabaseName { get; } = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+        public string DatabaseName { get; } = //"scratch";
```

Comment:
Is it possible to have a boolean flag in this class that switches between using/not using a scratch DB? It looks like people need to remember to also comment out other code if using a scratch DB, so using the boolean there to prevent errors in the future would be better too IMO. Seems like it would also remove the need for the dummy await below.